pkg/apis/rollouts/v1alpha1/types.go

```diff
@@ -77,8 +77,10 @@ type BlueGreenStrategy struct {
 	// Name of the service that the rollout modifies as the preview service.
 	// +optional
 	PreviewService string `json:"previewService,omitempty"`
-	// Steps define the order of steps to execute the bluegreen deployment
+	// ScaleDownDelay adds a delay before scaling down the previous replicaset. See
+	// https://github.com/argoproj/argo-rollouts/issues/19#issuecomment-476329960 for more information
 	// +optional
+	ScaleDownDelay *int32 `json:"scaleDownDelay,omitempty"`
```

Comment:
I would rename it to `ScaleDownDelaySeconds` for clarity (  other k8s resource field follows that conversion too e.g. Deployment ProgressDeadlineSeconds, MinReadySeconds )

Reply:
👍